src/main/features/dashboard/views/macro/claimStatus/offerSettlement.njk

```diff
@@ -113,7 +113,7 @@
     </p>
 
     <p> {{ t('When you’ve been paid in full, you need to let us know.') }} </p>
-    <a href="{{ AppPaths.contactUsPage.uri }}" class="button button-secondary">Tell us you've been paid</a>
+    <a href="{{ DashboardPaths.datePaidPage.evaluateUri({ externalId: claim.externalId }) }}" class="button button-secondary">Tell us you've been paid</a>
```

Comment:
as this change impacts production please do not merge these dashboard changes until you get sign off from QA that it is fully tested and Marc has ok'd it to go live